pkg/volume/glusterfs/glusterfs.go

```diff
@@ -200,10 +200,7 @@ func (plugin *glusterfsPlugin) ConstructVolumeSpec(volumeName, mountPath string)
 	glusterfsVolume := &v1.Volume{
 		Name: volumeName,
 		VolumeSource: v1.VolumeSource{
-			Glusterfs: &v1.GlusterfsVolumeSource{
-				EndpointsName: volumeName,
-				Path:          volumeName,
-			},
+			Glusterfs: &v1.GlusterfsVolumeSource{},
```

Comment:
I think maybe just return error since we cannot reconstruct volume spec for glusterfs.

Reply:
@jingxu97 I can get the `path` filled by proper volume name , however reconstructing `endpoint` from mount string looks to be impossible. I can return error , hopefully that can cause the reconciler to start unmount process.